schema/generator.go

```diff
@@ -1512,6 +1552,18 @@ func (g *Generator) haveSameColumnDefinition(current Column, desired Column) boo
 		reflect.DeepEqual(current.comment, desired.comment)
 }
 
+func (g *Generator) areSameGenerated(generatedA, generatedB *Generated) bool {
+	if generatedA == nil && generatedB == nil {
+		return true
+	}
+	if generatedA == nil || generatedB == nil {
+		return false
+	}
+	// TODO: Difference between bracketed and unbracketed, as Expr values are not fully comparable.
+	return (generatedA.expr == generatedB.expr || generatedA.expr == "("+generatedB.expr+")") &&
+		generatedA.generatedType == generatedB.generatedType
```

Comment:
TODO
Defining a lighter expression without parentheses, such as A / B * C instead of (A / B) * C in Expr, makes a difference every time.